### DISCUSSION TITLE
Update v6.6 - dev.6

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#211] [Pre-Release]
* fix tag rename crash
* fix `OrderListPreferenceDialog` and `PleaseWaitDialog` button bar
* add preview for icon shapes
* fix empty tags menu

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.6-dev.6'>Update v6.6 - dev.6</a>.</em>

**Comment by ltguillaume** [comment]
1. I think the tag renaming issue is completely solved, thanks!
2. The Tag Manager doesn't take into account the set order (which is not a biggie, but I don't know if you'd like to have that). Alternatively, you could allow the user to set the tag order using drag and drop within the Tag Manager, so that there isn't a need for yet another "deep level" settings dialog for that.
3. Shape previews look good.
4. Haven't had a completely empty tags menu, so can't confirm it's fixed.
